aviatrix/resource_aviatrix_cloudn_transit_gateway_attachment.go

```diff
@@ -80,6 +80,18 @@ func resourceAviatrixCloudnTransitGatewayAttachment() *schema.Resource {
 				Default:     true,
 				Description: "Enable dead peer detection.",
 			},
+			"enable_learned_cidrs_approval": {
```

Comment:
There should be default value for a bool attribute.

Reply:
Do we need default value for bool when it is false? I thought it is implied that default is false for optional values.